### ISSUE TITLE
Anisya: Debugging

### ISSUE THREAD

**Issue by AnisyaPurnama** [HackYourFutureBelgium/class-11-12#146]
> [module repository](https://github.com/HackYourFutureBelgium/debugging#getting-started) 

## Suggested Study

- [ ] study.hackyourfuture.be/javascript   (Watch all the 'Videos' one, still need to study the rest)

- [x] compute-it.toxicode  (it's fun! 😸) 

- [x] What is JavaScript?

- [ ] Debugging Tools

## Getting Started
- [x] Install the study-lenses package globally
- [x] Clone the repository
- [x] Open the repository using study

---

## Week 1

- [x] Sunday Prep

- [x] Sunday Class

### Check-In

__Progress__
Still struggling with the `study-lenses` exercise. 🤯  Well I'm quite slow at it, need some time until I solve each exercise. In the mean time I read/watch the suggested study and continuing the progress of my portfolio.

__Blocked__

__Next Steps__
Continue with the `study-lenses` exercise.

__Tip(s) of the week__
Do the suggested study as much as possible.

---

**Comment by colevandersWands** [comment]
Hi there @AnisyaPurnama , slow is fine!  Learning JavaScript is not a race.  Are you becoming comfortable with the studying/debugging workflow? stepping through and predicting your code's behavior

**Comment by AnisyaPurnama** [comment]
@colevandersWands yes, I'm quite comfortable and try to following the workflow. Since the js will be one of the most important language, I really want to understand it. And you are super right,  learning JavaScript is not a race. I really need to understand all the basic first before it become an advance level.

**Comment by colevandersWands** [comment]
@AnisyaPurnama , `And then maybe try another exercise platform.`  Let me know when you reach this point and I can help you find a good one for you.   In the next module we'll also start studying CodeWars which might be what you're looking for
